Jenkinsfile

```diff
@@ -0,0 +1 @@
+ezcaterPipeline("v1.3")
```

Comment:
I recommend dropping this as we don't use our internal build pipeline for open source projects.